packages/manager/src/features/LoadBalancers/LoadBalancerLanding/LoadBalancerEmptyStateData.ts

```diff
@@ -0,0 +1,71 @@
+import type {
+  ResourcesHeaders,
+  ResourcesLinkSection,
+  ResourcesLinks,
+} from 'src/components/EmptyLandingPageResources/ResourcesLinksTypes';
+
+export const headers: ResourcesHeaders = {
+  description:
+    'Scalable Layer 4 . and Layer 7 load balancer to manage and accelerate enterprise traffic between clients and your distributed applications and networks globally.',
```

Comment:
Here is the latest copy

```suggestion
    'Scalable Layer 4 and Layer 7 load balancer to route and manage enterprise traffic between clients and your distributed applications and networks globally.',
```